src/utils/HealthLogger.js

```diff
@@ -200,6 +202,35 @@ define(function (require, exports, module) {
         setHealthDataLog("searchDetails", searchDetails);
     }
 
+    /**
+     * Send Analytics Data
+     * @param {string} eventCategory The kind of Event Category that needs to be logged- should be a js var compatible string
+     * @param {string} eventSubCategory The kind of Event Sub Category that needs to be logged- should be a js var compatible string
+     * @param {string} eventType The kind of Event Type that needs to be logged- should be a js var compatible string
+     * @param {string} eventSubType The kind of Event Sub Type that needs to be logged- should be a js var compatible string
+     */
+    function sendAnalyticsData(eventName, eventCategory, eventSubCategory, eventType, eventSubType) {
```

Comment:
make optional parameters if possible 
i.e.
eventSubType could be initialized with ""
`eventParams.eventCategory = eventCategory || ""`
So you don't need to send empty string every time